src/server/scripts/Pet/pet_dk.cpp

```diff
@@ -168,9 +168,9 @@ class npc_pet_dk_ebon_gargoyle : public CreatureScript
                 _initialSelection = false;
                 // Find victim of Summon Gargoyle spell
                 std::list<Unit*> targets;
-                Acore::AnyUnfriendlyUnitInObjectRangeCheck u_check(me, me, 50);
+                Acore::AnyUnfriendlyUnitInObjectRangeCheck u_check(me, me, 30.0f);
```

Comment:
Why lowered that distance? Is it related to this commit at all?

Reply:
A error was being thowing up durning the compile testing. I aligned the distance with what tc had and it was shown as a double to float.

Reply:
I'm talking about `.f` - talking about distance. You changed from 50 yards to 30 yards - why?

Reply:
It was throwing a double to float error. Now i could of done it as 50.0f, but this just didnt seem to be consistent with other cores i worked on before. If it needs to be set to 50.0f, then the adjustment wouldnt be a issue in this case to change if you feel like it needs to be set to 50.

Reply:
Yes- change it back to 50.f